mentoring/choosing_a_solution.md

```diff
@@ -1,79 +1,104 @@
-So click on that solution and enter the Mentoring Discussion UI!
+Post any questions in the discussion screen to get clarification on the solution.
 
-## The Mentoring Discussion UI
+## The Student
+If you have a particular student you'd like to mentor, search for the student using the filter box, order by oldest or most recent.  
 
-At this stage you're looking at someone's solution, but haven't yet committed to mentoring it.
-You first have the opportunity to read their code and get some other information, before starting.
+On the right-hand side of the screen, filter by language or exercise.  
+
+You can also choose to only show exercises you've completed yourself (enabled by default). This is a great starting point until you are comfortable with higher level exercises.
+
+The main table shows the exercise, the student and when they requested mentoring.
+Hovering over a row gives you more detail about the user:  
+- Name
+- Location
+- Reputation (an indicator if they are a contributor or mentor themselves)
+- How many times they've been mentored before   
+
+The information on the tooltip is a preliminary indicator whether this student is a right match for you.
+You'll see information they've written explaining what they're looking to get out of the track. 
+
+As you start mentoring students, you'll see whether you've mentored them before, and if you've added them as a favorite.
 
-**If you're new to this UI, it might feel a little overwhelming as there is a lot of information, but don't worry - it'll soon become familiar.**
 
 ### The student's code
 
-On the left hand side of the screen you will see the student's code.
-The UI will look something like this:
+On the left hand side of the screen you will see the student's code.  
+
+The screen looks like this:
 
 [TODO: Image of mentor discussion with areas marked].
 
-1. The main part of the left side contains the student's code.
+1. The main part on the left side contains the student's code.
    By default you'll be seeing their most recent iteration.
    If they've submitted multiple iterations, you can switch through these using the numbers in circles at the bottom.
    If they've only submitted one iteration, you won't see those icons.
 
-2. You can use the tabs at the top to switch between the student's code, the tests, and the instructions.
+2. The tabs at the top shows the student's code, the tests, and the instructions.
    This is useful for reminding yourself about what the student has been asked to do in this exercise.
 
-3. You'll also see an indicator as to whether the tests have passed or failed.
-   If they failed, clicking on this indicator will open a modal that shows you the specific details of their test run, so you can see what they did wrong.
+3. There is an indicator showing whether a test has passed or failed.
+   If they failed, click the indicator to open a report showing the specific details of their test and what they did wrong.
```

Comment:
```suggestion
   If the tests have failed, click the indicator to open a report showing the details of the test and what is wrong.
```

Removing the personal attribution of failure (even though I know it isn't an attack, people do tend to get tied to their code personally, and this helps to remove the accusatory tone attributed to the user, and places it on the tests as a fact).

Also details are specific, so it may be redundant to generally state "the" when referencing details, while it would be appropriate if we were talking about a very specific detail.